README.md

```diff
@@ -250,6 +250,13 @@ In clearlinux based image, it is 2019.3 - to be upgraded later soon.
 Alternatively unit tests can be executed in a container by running the script `./tests/scripts/unit-tests.sh`
 
 `make test` to execute full set of functional tests (it requires [building the docker image](README.md#building) in advance). 
+=======
+`make test` to execute functional tests (it requires building the docker image in advance). Running 
+the tests requires also preconfigured env variables `GOOGLE_APPLICATION_CREDENTIALS`, `AWS_ACCESS_KEY_ID`,
```

Comment:
I thought we drop the google credentials? @joannanosek you have some information about this?

Reply:
Correct, google credentials are not needed anymore.

Reply:
Also the formatting of README [here](https://github.com/rasapala/OpenVINO-model-server) of this fragment looks awful :(